### ISSUE TITLE
suggestion: move talk details to a detail page

### ISSUE THREAD

**Issue by smeijer** [kentcdodds/remix-kentcdodds#85]
We currently show all details on the `/talks` page. While feel like it improves "scannability", it is a LOT of information to digest. I think we're better off moving details to a detail page. Click from `/talks` to `/talks/event-x` which then shows things like all tags, the presentations, and the recording.

That gives more space than we have now, and thereby would for example also make it possible to embed the youtube recording and/or slides, while the overview page would feel less crowded.

**Comment by kentcdodds** [comment]
I like that idea 👍 Let's do it.

**Comment by kentcdodds** [comment]
Yeah, still want to do this I think. I'm thinking it could be a bit like the calls page.
